android/src/main/java/com/revenuecat/purchases/react/RNPurchasesModule.java

```diff
@@ -318,6 +320,29 @@ public void setCreative(String creative) {
         SubscriberAttributesKt.setCreative(creative);
     }
 
+    @ReactMethod
+    public void canMakePayments(ReadableArray features, final Promise promise) {
+      ArrayList<Integer> featureList = new ArrayList<>();
+      if (features != null) {
+        for (int i = 0; i < features.size(); i++) {
+          featureList.add(features.getInt(i));
+        }
+      }
```

Comment:
You can do `features.toArrayList()` instead

Reply:
we can't since ReadableArray isn't typed :/ would have to convert everything to integers after anyway

Reply:
Ohhh I see 😞